app/client/components/cancel/membership/membershipCancellationReasons.tsx

```diff
@@ -39,29 +30,44 @@ export const membershipCancellationReasons: CancellationReason[] = [
         <SwitchToContributionPlaceholder />
       </>
     ),
-    experimentTriggerFlag: "showSwitchToContributionPlaceholder",
-    alternateFeedbackThankYouBody:
-      "One of our customer service specialists will be in touch shortly."
+    experimentTriggerFlag: "showSwitchToContributionPlaceholder"
+    // alternateFeedbackThankYouBody: "One of our customer service specialists will be in touch shortly."
```

Comment:
Do we still need this comment here?

Reply:
yeah because these will be restored once Corona is over. I chose to comment rather than remove (and rely on VCS) because I want them to be more prominent.